cli/info/info.go

```diff
@@ -46,6 +54,13 @@ func infoAction(c *cli.Context) (err error) {
 			fmt.Fprintln(os.Stderr, err)
 			return err
 		}
+		if pretty {
+			if p, err := PrettyPrinter(resp).PrettyBlock(); err == nil {
+				resp = p // replace resp if pretty success
+			} else {
+				fmt.Fprintln(os.Stderr, "Output origin resp due to PrettyPrint fail: ", err)
```

Comment:
Suggest: "Output origin" -> "Fallback to original"

Reply:
Done.